dds/DCPS/RTPS/RtpsDiscoveryConfig.cpp

```diff
@@ -32,15 +32,15 @@ RtpsDiscoveryConfig::resend_period() const
   // see RTPS v2.1 9.6.1.4.2
   return TheServiceParticipant->config_store()->get(config_key("RESEND_PERIOD").c_str(),
                                                     TimeDuration(30 /*seconds*/),
-                                                    DCPS::ConfigStoreImpl::Format_IntegerSeconds);
+                                                    DCPS::ConfigStoreImpl::Format_IntegerMilliseconds);
```

Comment:
`Format_FractionalSeconds`? I think `Format_IntegerMilliseconds` would make all existing usage of `ResendPeriod` much much shorter.